### ISSUE TITLE
Error with auto updater in Linux when running Sonarr and Radarr

### ISSUE THREAD

**Issue by onedr0p** [galli-leo/Radarr#86]
- [ ] `Error occurred while executing task ApplicationUpdate: Startup folder not writable by user 'radarr'`

which I fixed by issuing: `sudo chown -R radarr:radarr /usr/lib/radarr/`

- [ ] `Error occurred while executing task ApplicationUpdate: Access to the path "/tmp/nzbdrone_backup/config.xml" is denied.`

we'll need to update `BackupService.cs` and provide a new identity instead of `nzbdrone_backup`

**Comment by AeonLucid** [comment]
I thought I replied to this in the previous issue but apparently I didn't

For the update to work, the user that is installing Radarr for the first time has to give permissions manually, that's just how it works.

For your second error, this was fixed some time ago.
See: https://github.com/galli-leo/Radarr/commit/d62dbd48ae60a7d878f24fba04e1d4ff561edbd2#diff-82dda069ae275373e5dbc01c1d976f43

You'll have to update to a new version to get rid of the error.
